ods_ci/tests/Tests/400__ods_dashboard/420__model_serving/420__model_serving.robot

```diff
@@ -119,6 +120,12 @@ Verify Tensorflow Model Via UI
     Run Keyword And Continue On Failure  Wait Until Keyword Succeeds  5 min  10 sec  Verify Serving Service
     Verify Model Status    ${MODEL_NAME}    success
     Set Suite Variable    ${MODEL_CREATED}    True
+    # If running on self-managed, fetch the CA bundle for the cluster
+    # so it can be used when making the inference request
+    ${self_managed} =    Is RHODS Self-Managed
+    IF  ${self_managed}==${TRUE}
+        Fetch Openshift CA Bundle
+    END
```

Comment:
we usually avoid putting conditions in the test itself. Could this be moved outside? maybe in the setup? 
Alternatively, `Fetch Openshift CA Bundle` kw can be implemented in the helpers.py so can be used directly in the send_random_inference_request

Reply:
>Could this be moved outside? maybe in the setup?
This can be done

>Fetch Openshift CA Bundle kw can be implemented in the helpers.py so can be used directly in the send_random_inference_request
I don't like this idea because I don't think send_random_inference_request should be fetching the CA as well. The logic in that function already account for the CA if it exists, otherwise it functions as it always has